docs_2.0/manual_EN/4.ngql-guide/12.vertex-statements/4.delete-vertex.md

```diff
@@ -0,0 +1,19 @@
+# DELETE VERTEX
+
+```nGQL
+DELETE VERTEX <vid1>[, <vid2>...]
+```
+
+Use `DELETE VERTEX` to delete vertices and the related incoming and outgoing edges of the vertices. The `DELETE VERTEX` statement deletes one vertex or multiple vertices at a time. You can use `DELETE VERTEX` together with pipe. For more information on pipe, see [Pipe operator](../5.operators/4.pipe.md).
+
+## Examples
+
+```ngql
+nebula> DELETE VERTEX "team1";
+
+nebula> GO FROM "player100" OVER serve YIELD serve._dst AS id | DELETE VERTEX $-.id;
+```
+
+The preceding query deletes the vertex whose ID is "team1".
```

Comment:
For the GO statement, we must tell readers that serve._dst is "team1". Otherwise, we cannot give the conclusion that "team1" is deleted for the GO statement?

Reply:
The preceding queries delete ...

this refers to both the "DELETE" line and the "GO" line, right?